### ISSUE TITLE
Amount to pay $0.01

### ISSUE THREAD

**Issue by JRafael91** [Enngage/ngx-paypal#178]
the amount to pay is not what I declare in the settings.

![image](https://user-images.githubusercontent.com/39107286/120357752-a1d92400-c2ba-11eb-84a2-2792bddbe698.png)

![image](https://user-images.githubusercontent.com/39107286/120357555-6cccd180-c2ba-11eb-8e6c-092c7b562052.png)

**Comment by Enngage** [comment]
Can you try with different Currency? E.g. EUR? I see no reason this wouldn't work and is probably not related to this library. Maybe PayPal / your account has some restrictions. Please try asking PayPal support as to why your value is not what you set it to be.

**Comment by Enngage** [comment]
Can you try with different Currency? E.g. EUR? I see no reason this wouldn't work and is probably not related to this library. Maybe PayPal / your account has some restrictions. Please try asking PayPal support as to why your value is not what you set it to be.

**Comment by JRafael91** [comment]
I solved it by downgrade the version 7.0.0 to 6.1.0

**Comment by Enngage** [comment]
@JRafael91 I don't think that the solution... Based on npm-stat this package had `184k` download since the version 7 was released and it would strange you are the only one having this issue :-) Maybe there was something wrong in the code?
